tools/build_arrow.sh

```diff
@@ -113,7 +113,7 @@ make -j$NPROC
 make install
 
 cd java
-mvn clean install -P arrow-jni -pl gandiva,c -am -Darrow.cpp.build.dir=$ARROW_INSTALL_DIR/lib -DskipTests -Dcheckstyle.skip
+mvn clean install -P arrow-jni -pl dataset,gandiva,c -am -Darrow.cpp.build.dir=$ARROW_INSTALL_DIR/lib -DskipTests -Dcheckstyle.skip
```

Comment:
This one will just install in ~/.m2/repository, but maybe you need to add this jar in pom.xml

Reply:
Added the dependency in pom.xml.